src/include/config.h

```diff
@@ -236,6 +237,7 @@ extern const char __WT_CONFIG_CHOICE_key_consistent[];
 extern const char __WT_CONFIG_CHOICE_key_out_of_order[];
 extern const char __WT_CONFIG_CHOICE_last_checkpoint[];
 extern const char __WT_CONFIG_CHOICE_le[];
+extern const char __WT_CONFIG_CHOICE_live_restore_clean_up[];
```

Comment:
I am surprised with this change, it does not appear it in the [original ticket](https://github.com/wiredtiger/wiredtiger/commit/4247228a571d7a813cf2f90020a63b8082e8e3c3#diff-874213cd17a05904cb2c05179ff449f04d8b2135221395d421a524df56c719f7)

Reply:
I think it's not, @whoisong, please try to compare with the original changes and stick to these. I feel that the `git cherry-pick` command took too much due to conflicts it automatically resolved.

Reply:
This is not expected, live restore is a new feature that hasn't been introduced in v8.0 yet